apps/teams-test-app/src/components/SharingAPIs.tsx

```diff
@@ -6,11 +6,11 @@ import BoxAndButton from './BoxAndButton';
 
 const SharingAPIs = (): ReactElement => {
   const [shareWebContentRes, setShareWebContentRes] = React.useState('');
+  const [capabilityCheckRes, setCapabilityCheckRes] = React.useState('');
+
+  const shareWebContent = (input: string): void => {
+    const shareRequest: sharing.IShareRequest<sharing.IShareRequestContentType> = JSON.parse(input);
 
-  const shareWebContent = (): void => {
-    const shareRequest: sharing.IShareRequest<sharing.IURLContent> = {
-      content: [{ type: 'URL', url: 'https://bing.com' }],
-    };
     const callback = (err?: SdkError): void => {
       if (err) {
         setShareWebContentRes(JSON.stringify(err));
```

Comment:
Is the callback invoked (with `err` being `undefined`) when the flow succeeds? If so, we should probably add some 'completed' message or something in that scenario and assert on that. And if it isn't, we might want to talk to Helen and change the design. Otherwise when we move the API to Promise<void> it'll never resolve.

Reply:
callback is not invoked in case of success. Let me synch with Helen about this.

Reply:
> It calls three other APIs with the callback.

Can you say more about that? What APIs are invoked instead?